README.md

```diff
@@ -1,13 +1,25 @@
+# An Introduction to Encryption and Learning Cryptography
+
+Cryptography is essential for security. 
```

Comment:
Who do you think is the target audience that reads this, and what should we communicate to them?

Thinking about the target audience: it's unlikely that anyone would attempt to learn cryptography from this paragraph, so I'd just use it as a justification for having this repo.

What do you think about simplifying this paragraph by keeping it short and sweet? My proposal is to have a sentence saying that cryptography is widely used today and provide a few bullet list of key problems it solves without going into much detail. We can also add a header "Why learn about cryptography" to make the intent of the paragraph clear. 

What do you think?

Reply:
Sure. Your point makes sense, I went on to add what I know about crypto which was slightly unlike the repo's intent. I have made the changes. Kindly review.